tests/contrib/django/testapp/tasks.py

```diff
@@ -0,0 +1,11 @@
+from ..testapp.celery import app
```

Comment:
That's a relative import. But I can change to `tests.contrib.django.testapp.celery` if you prefer